bin/mkosi

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/bash
+
+PYROOT=$(dirname "$0")
```

Comment:
Maybe add a comment here at the top to explain all the stuff we do in this script and which use cases we're trying to make work?